src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/EditorHomePageObject.java

```diff
@@ -25,6 +28,8 @@
   private WebElement addNewSectionButton;
   @FindBy(css = "section:nth-of-type(3) .curated-content-editor-add-item-btn")
   private WebElement addCategoryButton;
+  @FindBys(@FindBy(css = "section:nth-of-type(1) .curated-content-editor-row"))
+  private List<WebElement> featuredContentItem;
```

Comment:
is this featured content or section? Name is misleading compared to selector and method clickOnSectionByIndex